apps/desktop/src/components/settings/template/index.tsx

```
@@ -1,35 +1,109 @@
   const [currentTemplate, setCurrentTemplate] = useState<string | null>(null);
+  const [searchQuery, setSearchQuery] = useState("");
+  const [filterStatus, setFilterStatus] = useState<FilterStatus>("all");
   const templates = useTemplates();
 
+  const filteredTemplates = useMemo(() => {
+    let filtered = Object.entries(templates);
+
+    // Apply favorite filter (placeholder for now)
+    if (filterStatus === "favorite") {
+      // TODO: Implement favorite filtering when favorite field is added
+      filtered = filtered;
```

Biome (2.1.2)

[error] filtered is assigned to itself.

This is where is assigned.

Self assignments have no effect and can be removed.

(lint/correctness/noSelfAssign)